tic-tac-toe/src/components/board/board.js

```diff
@@ -0,0 +1,30 @@
+import React, {Component} from 'react';
+import Square from '../square/square';
+import './board.css';
+
+class Board extends Component {
+    renderSquare(i) {
```

Comment:
I don't see real reason to create this method, use it right in your map, it's even harder to read it now because you use index `i` and it's a bad name

Reply:
I rewrote this component and made.
```suggestion
function Board({squares, onClick}) {
  const squareComponents = squares.map((square, index) => {
    return (
      <Square
        value={squares[index]}
        onClick={() => {onClick(index)}}
        key={index}
      />
    )
  });   

  return (
    <div className={classes.Board}>
      {squareComponents}
    </div> 
  )
}
```